### PR TITLE
add: typography component

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

**Review by VishalPatil18 — APPROVED** [review]
The code looks clean as well as works great. Well done Savita 👍

File: components/typography/typography.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        <nav class="nav-bar">
+            <div class="left-nav">
+                <a href="#">
+                    UI Book
+                </a>
+            </div>
+            <button href="#" class="toggle-btn">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
```
Comment (ruchiragoswami) [inline components/typography/typography.html:27]: Not sure if the content in the span tag is forgotten or it is intentionally left blank. Please take a look.

**Review by ruchiragoswami — APPROVED** [review]
Looks great! I left a comment, please read it before merging.

File: components/typography/typography.html (hunk 1 above)
Comment (vermasavita) [reply components/typography/typography.html:27]: Yes, I did that intentionally for menu bar when screen will be shorter

**Pull request merged**